docs/src/operate/c8y/cloud-profiles.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cloud Profiles
+tags: [Operate, Cumulocity, Cloud Profile]
+description: Connecting %%te%% to multiple Cumulocity tenants
+---
+
+import UserContext from '@site/src/components/UserContext'; import
+UserContextForm from '@site/src/components/UserContextForm';
+
+**%%te%%** now supports multiple Cumulocity connections using cloud profiles.
```

Comment:
*Now* is vague.

Either:

```suggestion
**%%te%%** supports multiple Cumulocity connections using cloud profiles.
```

or :

```suggestion
Starting with version 1.4.0, **%%te%%** supports multiple Cumulocity connections using cloud profiles.
```

I would go with the second suggestion, as the docs are already versioned.

Reply:
I would also go with the second option.